resources/default_pipeline_environment.yml

```diff
@@ -327,8 +327,14 @@ steps:
     logSuccessfulMavenTransfers: false
   mtaBuild:
     buildTarget: 'NEO'
+    buildTool: classic
+    platform: 'cf'
     mtaJarLocation: '/opt/sap/mta/lib/mta.jar'
-    dockerImage: 'ppiper/mta-archive-builder'
+
+    classic:
+      dockerImage: 'ppiper/mta-archive-builder'
+    cloudMBT:
+      dockerImage: 'devxci/mbtci'
```

Comment:
we might want to fix the default to a certain tag (which needs to be updated from from time to time) just to avoid accidental breaks..

Reply:
Yes, we should do once image is finalized.

Reply:
change to v1
```suggestion
      dockerImage: 'devxci/mbtci:1.0.0'
```